tests/integration/payment/test_forms.py

```diff
@@ -11,6 +11,8 @@
 #               bankcards.DISCOVER, bankcards.JCB, bankcards.CHINA_UNIONPAY,
 #               bankcards.SOLO, bankcards.LASER, bankcards.SWITCH]
 
+valid_year = str(datetime.date.today().year)
```

Comment:
Could I request that you use the [same approach](https://github.com/django-oscar/django-oscar/blob/master/tests/integration/payment/test_forms.py#L49) that was used to fix this on master, just for consistency of approach? Using freezegun is also more deterministic.